identity/handler.go

```diff
@@ -413,6 +416,40 @@ type AdminUpdateIdentityBody struct {
 	State State `json:"state"`
 }
 
+// swagger:parameters adminPatchIdentity
```

Comment:
It would be good to change the order here so that the models are just before their handler.

Reply:
Right, done.